pkg/kwokctl/runtime/compose/cluster.go

```diff
@@ -285,6 +285,9 @@ func (c *Cluster) Install(ctx context.Context) error {
 	if err != nil {
 		return err
 	}
+	if conf.KwokControllerPort == 0 {
```

Comment:
https://github.com/kubernetes-sigs/kwok/blob/122f36ce87128371d2b9bbcff43837d764b35e37/pkg/kwokctl/components/prometheus.go#L67-L72

Using docker-compose requires exposing the port to the host here, just like Prometheus' port